tensorflow/stream_executor/rocm/rocm_helpers.cu.cc

```diff
@@ -31,5 +31,16 @@ void broadcast_fp32(void* stream, float* dst, int dst_stride, int batches, int s
   hipLaunchKernelGGL(broadcast_fp32_kernel, dim3(x_blocks, (batches+3)/4, src_batches), min(256, (int)size), 0,
                      (hipStream_t)stream, dst, dst_stride, batches, src, size);
 }
+__global__ void __xla_MakeBatchPointers(char* base, int stride, int n, void** ptrs_out) {
+  int idx = threadIdx.x + blockIdx.x * blockDim.x;
+  if (idx >= n) return;
+  ptrs_out[idx] = base + idx * stride;
+}
+
+void rocm_MakeBatchPointers(void* stream, char* base, int stride, int n, void** ptrs_out) {
+  hipLaunchKernelGGL(__xla_MakeBatchPointers, dim3((n + 256 - 1)/256, 1, 1), dim3(256, 1, 1), 0,
```

Comment:
explain what 256 is....change this to something like

```
int threads_per_block = 256;
int num_blocks = (batch_size + threads_per_block - 1) / threads_per_block;
```

Reply:
done.